app/middlewares/validators/users.js

```diff
@@ -1,54 +1,25 @@
+const { checkSchema, validationResult } = require('express-validator');
 const { User } = require('../../models');
 const errors = require('../../errors');
 
-exports.checkExistingEmail = async email => {
-  const user = await User.findOne({ where: { email } });
-
-  if (user) {
-    throw errors.conflictError('Trying to create a user that already exists');
-  }
-  return false;
-};
-
-exports.checkPasswordRestriction = password => {
-  const isAlphanumericRegex = /[a-zA-Z0-9]{8,61}/;
+const verifySchema = schema => checkSchema(schema);
```

Comment:
Put `verifySchema`, `validateResult ` and `validateBySchema` in a different file, since they'll be used for every schema and not only for user validations.